src/index.html

```diff
@@ -23,9 +23,55 @@
+  <div class="comment-pop-up-container">
+    <div class="image-thumbnail-delete-icon-container">
+      <img id='game-image' src="https://www.mmobomb.com/g/1/thumbnail.jpg" alt="game-image-thumbnail" class="comment-game-image">
+      <div id="delete-popup" class="comment-delete-icon"></div>
+        <!-- alt="delete-pop-up-icon"  -->
+    </div>
+    <h3 class="comment-name-of-game" id="name-of-game"> "Name of Game to Be Added Dynamically"</h3>
+    <div class="comment-game-description-details"> Short Description: <span id="short-description">"Short description to be filled in dynamically from server"</span></div>
+    <div class="comment-game-description-container">
+      <div class="comment-game-description-details"> Game developer: <span id="game-developer">"Short description to be filled in dynamically from server"</span></div>
+      <div class="comment-game-description-details"> Game Url: <span id="game-url">"Game Url to be filled in dynamically from server"</span></div>
+      <div class="comment-game-description-details"> Genre: <span id="genre">"Genre to be filled in dynamically from server"</span></div>
+      <div class="comment-game-description-details"> Platform: <span id="platform">"Platform to be filled in dynamically from server"</span></div>
+      <div class="comment-game-description-details"> Publisher: <span id="publisher">"Platform to be filled in dynamically from server"</span></div>
+      <div class="comment-game-description-details"> Release - date: <span id="release-date">"Release - date to be filled in dynamically from server"</span></div>
+    </div>
+    <div class="comment-comments-container">
+      <div class="comment-server-comments-container">
+        <h3 class="comment-server-comments-header-text"> COMMENTS ("number of comments to be shown from server")</h3>
+        <div class="comment-server-comments">
+          <p class="comment-individual-server-comment"> 
+            <span>"Time of the comment "</span> 
+            <span> "Author of the comment"</span> 
+            <span>"Actual Comment"</span> 
+          </p>
+          <p class="comment-individual-server-comment"> 
+            <span>"Time of the comment -2 "</span> 
+            <span> "Author of the comment - 2"</span> 
+            <span>"Actual Comment - 2"</span> 
+          </p>
+        </div>
+      </div>
+      <h3 class="comment-comments-form-header-text"> Add a Comment </h3>
+      <form id="comment-add-comment-form" class="comment-add-comment-form" action="submit">
+        <input type="text" placeholder="Your Name" id="commentorsName" class="comment-input-text-name"/>
+        <textarea placeholder="Your Comment" id="commentorsComment" class="comment-input-text-comment"></textarea>
+        <button type="submit" class="comment-button-submit"> COMMENT </button>
+      </form>
+    </div>
+  </div>
+</section>
+
+
+<!-- Main HomePage Section -->
   <main id="main">
     <h1 class="main-section-title">MMO Games</h1>
```

Comment:
- [ ] It is good to dynamically generate this from JavaScript upon comments button click event.